src/DomainServices/Core/UploadService.cs

```diff
@@ -15,8 +16,10 @@ class UploadService: IUploadService
         private UploadServiceOptions options;
         private BlobServiceClient blobServiceClient;
         private string blobContainerName="caff_container";
-        public UploadService(UploadServiceOptions options, string connectionString)
+        private ILogger logger;
+        public UploadService(ILogger logger, UploadServiceOptions options, string connectionString)
```

Comment:
Az ILogger vegye át template paraméterként az osztályt, ahogy korábban is kértem:
`ILogger<UploadService>`